server/block/sandstone_type.go

```diff
@@ -71,6 +71,26 @@ func (s sandstone) String() string {
 	panic("unknown sandstone type")
 }
 
+// Smooth varient
+func (s sandstone) Smooth() bool {
+	return s == 3
+}
+
+// Cut varient
+func (s sandstone) Cut() bool {
+	return s == 1
+}
+
+// Can be a Slab
+func (s sandstone) SlabAble() bool {
+	return s != 2
+}
+
+// Can be a Stairs
```

Comment:
Make sure to use a full sentence here with full stop and start it with the function name, so something like `SlabAble checks if a slab with this type can exist.` and `StairAble checks if a stair with this type can exist.`